gradle/binary_compatibility.gradle

```diff
@@ -15,7 +15,7 @@ buildscript {
 }
 
 File baselineJar = null
-def baselineVersion = "1.3.4"
```

Comment:
I think this is incorrect - what we'd like to do is to disable JAPICMP checking for 2.0, and keep doing it for 1.0. I think that we should in fact not really need a 2.0 branch, if we're putting the 2.x code 'next to' the 1.x code, so that they can coexist. 

Maybe a better/easier approach would be to simply add the 2.0 modules next to the 1.x modules, but disable publishing and JAPICMP checking for them until we're ready to start publishing?

Reply:
Hmm... I'm not really sure how to set that up, but this branch should in fact be able to live on master, I haven't touched any of the old modules. I figured it'd make sense to keep it separate for the time being but I don't really have a strong opinion on how to do this.

Reply:
Let's keep it on a separate branch.